org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -1483,40 +1483,6 @@ SsoAgoraDevDeveloper:
     principalId: !Ref agoraDevDeveloperGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-developer-permission-set-arn' ]
 
-SsoIncludeChopProdAdmin:
-  Type: update-stacks
-  DependsOn: SsoAdministrator
-  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.11/templates/SSO/aws-sso.yaml
-  StackName: !Sub '${resourcePrefix}-${appName}-includechopprod-admin'
-  StackDescription: 'SSO: admin role used by includechopprod admin group'
-  DefaultOrganizationBindingRegion: !Ref primaryRegion
-  DefaultOrganizationBinding:
-    IncludeMasterAccount: true
-  OrganizationBindings:
-    TargetBinding:
-      Account: !Ref IncludeChopProdAccount
-  Parameters:
-    instanceArn: !Ref instanceArn
-    principalId: !Ref includeChopProdAdminGroup
```

Comment:
`includeChopProdAdminGroup` is defined in this file so please remove that as well.